### ISSUE TITLE
Partial marks

### ISSUE THREAD

**Issue by va5hi** [nus-cs2030/2122-s1#345]
Are there partial marks for labs and project work that does not get A on codecrunch? I have a few that's not A and its really panic szn now.

**Comment by Devvynneo** [comment]
Yes according to some of the previous posts, deemed sufficient, you will be given partial marks

**Comment by seasonedlasagna** [comment]
In this comment, Prof has mentioned that partial grading will be given for both projects and labs

https://github.com/nus-cs2030/2122-s1/issues/149#issuecomment-955123155

**Comment by chenghong99** [comment]
Relax u still have tonight and the whole pf tmr to get the labs and project done. Jiayouss!!

**Comment by yuehernkang** [comment]
jiayou everyone !
